src/randoop/operation/Operation.java

```diff
@@ -4,62 +4,166 @@
+   * Returns the ordered list of input types for this operation.
+   * If a method call or field access, the first input corresponds to the
+   * receiver, which must be an object of the declaring class.
+   * 
+   * @return list of types as {@link Class} objects.
    */
   List<Class<?>> getInputTypes();
 
   /**
-   * Returns the type of this statement kind's output.
+   * getOutputTypes gives the type returned by the operation.
+   * 
+   * @return type returned by the {@link Operation} as a {@link Class} object.
    */
   Class<?> getOutputType();
 
-
   /**
-   * Executes this statement, given the inputs to the statement. Returns
+   * Performs this operation using the array of input values. Returns
    * the results of execution as an ResultOrException object and can
    * output results to specified PrintStream.
-   * @param statementInput array containing appropriate inputs to statement
+   * @param input array containing appropriate inputs to operation
    * @param out stream to output results of execution;
    *            can be null if you don't want to print.
    * @return results of executing this statement
    */
-  ExecutionOutcome execute(Object[] statementInput, PrintStream out);
-
+  ExecutionOutcome execute(Object[] input, PrintStream out);
 
   /**
    * Produces a Java source code representation of this statement and append it
    * to the given StringBuilder.
+   * 
+   * @param inputVars  the list of variables that are inputs to operation.
+   * @param b  the {@link StringBuilder} to which code is added.
    */
-  void appendCode(Variable newVar, List<Variable> inputVars, StringBuilder b);
+  void appendCode(List<Variable> inputVars, StringBuilder b);
 
   /**
-   * Returns a string representation of this StatementKind, which can
-   * subsequently be used in this class's parse method. For a class C
-   * implementing the StatementKind interface, this method should return a
-   * String s such that parsing the string returns an object equivalent to
-   * this object, i.e. C.parse(this.s).equals(this).
+   * Returns a string representation of this Operation, which can be read by 
+   * static parse method for class. 
+   * For a class C implementing the Operation interface, this method should 
+   * return a String s such that parsing the string 
+   * returns an object equivalent to this object, i.e. C.parse(this.s).equals(this).
+   * 
+   * @return string descriptor of {@link Operation} object.
    */
   String toParseableString();
 
+  /**
+   * Predicate to indicate whether object represents a static operation on the 
+   * declaring class.
+   * 
+   * @return true if operation corresponds to static method or field of a class, 
+   *         and false, otherwise.
+   */
+  boolean isStatic();
+
+  /**
+   * Predicate to indicate whether object represents a method-call-like 
+   * operation (either static or instance). 
+   * This include non-method operations that access fields.
```

Comment:
"include" => "includes".
I would be more specific:
"This includes field getters and setters."